spec/namespaces/flow_framework.yaml

```
@@ -0,0 +1,261 @@
+      parameters:
+        - $ref: '#/components/parameters/flow_framework.create::query.provision'
+        - $ref: '#/components/parameters/flow_framework.create::query.validation'
+
+      requestBody:
+        $ref: '#/components/requestBodies/flow_framework.create'
+      responses:
+        '201':
+          $ref: '#/components/responses/flow_framework.create'
+  /_plugins/_flow_framework/workflow/{workflow_id}:
+    delete:
+      operationId: flow_framework.delete.0
+      x-operation-group: flow_framework.delete
+      description: Delete a workflow.
+      externalDocs:
+        url: https://opensearch.org/docs/latest/automating-configurations/api/delete-workflow/
+      parameters:
+        - $ref: '#/components/parameters/flow_framework.delete::query.clear_status'
+      responses:
+        '200':
+          $ref: '#/components/responses/flow_framework.delete'
+        '403':
+          description: if workflow id is null.
+    put:
+      operationId: flow_framework.update.0
+      x-operation-group: flow_framework.update
+      description: Update a workflow. You can only update a complete workflow if it has not yet been provisioned.
+      externalDocs:
+        url: https://opensearch.org/docs/latest/automating-configurations/api/create-workflow/
+      parameters:
+        - $ref: '#/components/parameters/flow_framework.update::query.update_fields'
+        - $ref: '#/components/parameters/flow_framework.get::path.workflow_id'
+      requestBody:
+        $ref: '#/components/requestBodies/flow_framework.update'
+      responses:
+        '201':
+          $ref: '#/components/responses/flow_framework.create'
+          description: It returns the workflow_id
+    get:
+      operationId: flow_framework.get.0
+      x-operation-group: flow_framework.get
+      description: Get a workflow.
+      parameters:
+        - $ref: '#/components/parameters/flow_framework.get::path.workflow_id'
+      responses:
+        '200':
+          $ref: '#/components/responses/flow_framework.get'
+
+
+components:
+  parameters:
+    flow_framework.get::path.workflow_id:
+      name: workflow_id
+      in: path
+      required: true
+      schema:
+        type: string
+    flow_framework.create::query.provision:
+      in: query
+      name: provision
+      schema:
+        description: Whether to provision the workflow as part of the request
+        type: boolean
+        default: false
+    flow_framework.create::query.validation:
+      in: query
+      name: validation
+      schema:
+        description: Whether to validate the workflow. Valid values are all (validate the template) and none (do not validate the template). Default is all.
+        type: string
+        default: all
+    flow_framework.update::query.update_fields:
+      in: query
+      name: update_fields
+      schema:
+        description: Whether to update only the fields included in the request body.
```

```suggestion
        description: Only update the fields included in the request body.
```

Try to rewrite all the other `.... whether ....` by something more directed like above.